verilog/tools/kythe/kythe_facts_extractor.cc

```diff
@@ -959,6 +925,16 @@ VName KytheFactsExtractor::ExtractStructOrUnion(
   return struct_vname;
 }
 
+void KytheFactsExtractor::CreateAnchorReferences(
+    const std::vector<Anchor>& anchors,
+    const std::vector<const VName*>& definitions) {
+  // Loop over the definitions and create kythe facts.
+  for (size_t i = 0; i < definitions.size(); i++) {
```

Comment:
Would this be a good spot to verify size-equality of vectors?

Reply:
No, we don't actually care about size here. 
Each `i` reference from anchors should map to `i` from the definitions and if the sizes aren't equal we just loop over the size of the definitions and its size `<=` anchor.size().